src/bundle/Resources/views/themes/admin/content/tab/url/custom_urls_table.html.twig

```diff
@@ -45,7 +45,7 @@
                 <tr>
                     <td class="ez-table__cell ez-table__cell--no-content" colspan="4">
                         <p class="ez-table-no-content mb-0 py-0">
-                            {{ 'tab.urls.no_custom_urls'|trans|desc('This Content item has no custom URL aliases.') }}
+                            {{ 'tab.urls.no_custom_urls'|trans|desc('This Content item has no custom URL aliases or they are broken.') }}
```

Comment:
Pass this message to people handling microcopy - I'm expecting them to not be happy about always suggesting that some links are broken.